Edits/azure-service-bus-management-connector-studio.adoc

```diff
@@ -0,0 +1,135 @@
+= Use Studio to Configure Azure Service Bus Management 1.0 - Mule 4
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure an input source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-other-fields,Configure the other connector fields>>.
+
+When you run the connector, you can view the app log. For information, see <<view-app-log,View the App Log>>.
+
```

Comment:
The following is a GLOBAL change to the template; can you change it there as well as here?